TodoApi/Extensions/OpenTelemetryExtensions.cs

```diff
@@ -23,15 +22,16 @@ public static WebApplicationBuilder AddOpenTelemetry(this WebApplicationBuilder
         var resourceBuilder = ResourceBuilder.CreateDefault().AddService(builder.Environment.ApplicationName);
         var oltpEndpoint = builder.Configuration["OTEL_EXPORTER_OTLP_ENDPOINT"];
 
-        builder.Logging.AddOpenTelemetry(logging =>
-        {
-            logging.SetResourceBuilder(resourceBuilder);
-
-            if (!string.IsNullOrWhiteSpace(oltpEndpoint))
-            {
-                logging.AddOtlpExporter();
-            }
-        });
+        //// Using serilog for logging
+        // builder.Logging.AddOpenTelemetry(logging =>
+        // {
+        //     logging.SetResourceBuilder(resourceBuilder);
+        //
+        //     if (!string.IsNullOrWhiteSpace(oltpEndpoint))
+        //     {
+        //         logging.AddOtlpExporter();
+        //     }
+        // });
```

Comment:
I would change this to:

```suggestion
if (!string.IsNullOrWhiteSpace(oltpEndpoint))
{
    builder.Logging.AddOpenTelemetry(logging =>
    {
        logging.SetResourceBuilder(resourceBuilder)
               .AddOtlpExporter();
    });
}
```

Reply:
Did you apply this change? I don't see it in the PR?